backend/ibutsu_server/openapi/openapi.yaml

```
@@ -1483,7 +1483,7 @@ components:
           xfailures: 0
           xpasses: 1
           tests: 548
-        metadata:
+        data:
```

Though this is called `data` in the DB model, in Ibutsu's API and FE we still refer to it as `metadata`, and do the translation to `data` in the `ModelMixin`

https://github.com/ibutsu/ibutsu-server/blob/721a77db65f78d651effb0db35c53c487c5c9378/backend/ibutsu_server/db/models.py#L44-L45

```suggestion,
        metadata:
```

---

Ah, OK. I thought so, except the example said, "data" and not "metadata"